.github/workflows/deploy_to_aws.yml

```
@@ -143,7 +143,7 @@ jobs:
           node-version: '14'
 
       - name: Cache dependencies
-        uses: actions/cache@v4
+        uses: actions/cache@v2
```

actionlint (1.7.4)

the runner of "actions/cache@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)